src/components/Navbar/Navbar.scss

```diff
@@ -0,0 +1,42 @@
+.Navbar {
+  width: 90%;
+  height: 50px;
+  margin: 0;
```

Comment:
margin: 0 auto;
this way it will be centered and margins will be equal by both sides